rubrik_cdm/data_management.py

```diff
@@ -321,6 +321,14 @@ def object_id(self, object_name, object_type, host_os=None, hostname=None, share
             if hostname is None:
                 raise InvalidParameterException(
                     "You must provide the host or one of the hosts in a RAC cluster for the Oracle DB object.")
+            regex = '''^(25[0-5]|2[0-4][0-9]|[0-1]?[0-9][0-9]?)\.( 
```

Comment:
Can you add a comment that describes what exactly this regex is looking for?

Reply:
I will add the comment. It's to check if the variable is an IP address. The lookup for an oracle_db object does not support IP addresses so I use that to check so that a more accurate error can be raised.. This may be better placed into a function so it can be reused but I didn't want to add complication.